hibike/ATALANTA_INTEGRATION.md

```diff
@@ -0,0 +1,83 @@
+# Hibike Runtime Integration
+
+All hibike smart devices are abstracted as a mapping of parameters to values.
+Runtime can read from, write to, or subscribe to these parameters.
+Paramaters are always strings, but the type of each value is unique.
+A config file will describe the type of the value for each paramater, and the parameters for each smart device type.
+A parameter can be read only, write only, or read/write.
+
+## Example Smart Devices:
+
+LimitSwitch
+
+- switch0: bool read only
+- switch1: bool read only
+- switch2: bool read only
+- switch2: bool read only
+
+Servo
+
+- servo0_torqued: bool read/write
+- servo0_pos: int read/write
+
+MetalDetector
+
+- callibrate: bool write only
```

Comment:
Oh by the way @karthik-shanmugam, you and @yizow should figure out how to resolve the conflicts in this file and the json (his PR changed the formatting, fixed spelling of calibrate, etc.)